students/carlos_novoa/lesson04/mailroom.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+import os
+import sys
+import datetime
+
+"""
+Lesson4 - Mailroom, Part 2
+"""
+
+now = datetime.datetime.now()
+
+_donors = [
+    [
+        {
+            'first_name': 'Jim',
+            'last_name': 'Halpert'
+        },
+        {
+            'donations': [
+                1.00
+            ]
+        }
+    ],
+    [
+        {
+            'first_name': 'Pam',
+            'last_name': 'Beesley'
+        },
+        {
+            'donations': [
+                1000.00,
+                2000.00,
+                3000.00
+            ]
+        }
+    ],
+    [
+        {
+            'first_name': 'Dwight',
+            'last_name': 'Shrute'
+        },
+        {
+            'donations': [
+                2.00,
+                3.00
+            ]
+        }
+    ],
+    [
+        {
+            'first_name': 'Michael',
+            'last_name': 'Scott'
+        },
+        {
+            'donations': [
+                10.00,
+                20.00,
+                30.00
+            ]
+        }
+    ],
+    [
+        {
+            'first_name': 'Andy',
+            'last_name': 'Bernard'
+        },
+        {
+            'donations': [
+                500.00
+            ]
+        }
+    ]
+]
```

Comment:
you should use `dict` as a data structure for donors instead of a list, which is easier and more efficient in this use case